client/src/components/AlertConfigsTablePage.js

```diff
@@ -0,0 +1,143 @@
+                                    {
+                                        title: "Start Date (M/D/Y)",
+                                        field: "start_date",
+                                    },
+                                    {
+                                        title: "End Date (M/D/Y)",
+                                        field: "end_date",
+                                    },
+                                    {
+                                        title: "Max Weekly",
+                                        field: "max_weekly",
+                                    },
+                                    {
+                                        title: "ETA",
+                                        field: "eta",
+                                    },
+                                    {
+                                        title: "Arrival",
+                                        field: "arrival",
+                                    },
+                                    {
+                                        title: "Cancelled",
+                                        field: "cancelled",
+                                    },
+                                    {
+                                        title: "Departure",
+                                        field: "departure",
+                                    },
+                                    {
+                                        title: "Diverted",
+                                        field: "diverted",
+                                    },
+                                    {
+                                        title: "Filed",
+                                        field: "filed",
+                                    }
+                                ]}
+                                data={data.map(alert => (
+                                    {
+                                        fa_alert_id: alert.fa_alert_id,
+                                        ident: alert.ident,
+                                        origin: alert.origin,
+                                        destination: alert.destination,
+                                        aircraft_type: alert.aircraft_type,
+                                        start_date: (new Date(alert.end_date)).toLocaleDateString(),
+                                        end_date: (new Date(alert.end_date)).toLocaleDateString(),
+                                        max_weekly: alert.max_weekly,
+                                        eta: alert.eta,
+                                        arrival: alert.arrival,
+                                        cancelled: alert.cancelled,
+                                        departure: alert.departure,
+                                        diverted: alert.diverted,
+                                        filed: alert.filed
```

Comment:
For your consideration, but by no means a request: Instead of showing the true/false a readily used symbols might be easier to understand at a glance.
```suggestion
                                        arrival: (alert.arrival ? '✔' : '✖'),
                                        cancelled: (alert.cancelled ? '✔' : '✖'),
                                        departure: (alert.departure ? '✔' : '✖'),
                                        diverted: (alert.diverted ? '✔' : '✖'),
                                        filed: (alert.filed ? '✔' : '✖')
```